packages/blitz/src/cli/commands/install.ts

```diff
@@ -62,8 +65,9 @@ const requireJSON = (file: string) => {
   return JSON.parse(require("fs-extra").readFileSync(file).toString("utf-8"))
 }
 
-const checkLockFileExists = (filename: string) => {
-  return require("fs-extra").existsSync(resolve(filename))
+const checkLockFileExists = async (filename: string) => {
+  const dotBlitz = join(await findNodeModulesRoot(process.cwd()), ".blitz")
+  return require("fs-extra").existsSync(resolve(join(dotBlitz, "..", "..", filename)))
```

Comment:
Question: why "../../" ? I think we should start documenting these things, ad it's not clear why and where we do "../",  "../../", "../../../" etc.

Reply:
Using the same method in other places to find the .blitz directory, then we go up a couple directories to find the package lockfile. I feel this way works best, especially for monorepos.